lobid-vocab.ttl

```diff
@@ -659,6 +679,15 @@ lv:subjectAltLabel
    rdfs:domain dct:BibliographicResource ;
    rdfs:range rdfs:Literal .
 
+# <http://purl.org/lobid/lv#subjectsLabels>
+lv:subjectsLabels
+   rdf:type rdf:Property ;
+   rdfs:label "All labels of subjects"@en ;
+   rdfs:label "Alle Bennenungen der Schlagworte"@de ;
+   rdfs:isDefinedBy <http://purl.org/lobid/lv> ;
+   rdfs:comment "Contains labels which are used for the Verbundkatalog search but are not shown in the frontend."@en ;
+   rdfs:domain dct:BibliographicResource ;
+   rdfs:range rdfs:Literal .
```

Comment:
AFAIK, this field can be queried but it is not delivered via the API as it is only used kind of hbz-internally in the DigiBib. Thus, I suggest leaving it out of this vocabulary definition.